selfdrive/ui/qt/widgets/cameraview.cc

```diff
@@ -199,12 +199,13 @@ void CameraViewWidget::updateFrameMat(int w, int h) {
 }
 
 void CameraViewWidget::paintGL() {
-  if (latest_texture_id == -1) {
+  if (latest_texture_id == -1 || stream_type == VISION_STREAM_RGB_FRONT) {
```

Comment:
this isn't a real way to fix the issue. how expensive is it to always do this?

Reply:
It's very cheap and doesn't even make a difference

Reply:
We can only clear the background every frame, otherwise there will be problems on C2

Reply:
It should be caused by  by the difference between c2's GraphicBuffer/visonbuf and other devices

Reply:
I don't understand. is it not cheap to always glClearColor and glClear on all platforms for all cameras?

Reply:
it's same on all platforms. think we should always call glclear before drawing to avoid weird artifacts